src/utils/common.js

```diff
@@ -121,7 +121,7 @@ export const logBreadcrumb = (
     else printLog(`${level} - ${category}: ${message}`);
   }
   Sentry.addBreadcrumb({
-    category, message, level, data: { extra },
+    category, message, level, data: extra,
```

Comment:
My only concern would be here if `extra` is passed not as object and flow fails to read that what would happen then?

Reply:
The reason I've put it here, was that breadcrumbs have limited depth in Sentry, and adding another level of nesting actually limited displayed data in sentry. Will add some safety checks here.

Reply:
👍